exercises/intro_to_pages/config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Hello Page",
+    "name": "Intro to Pages",
```

Comment:
When I was working on this exercise, I was confused because I was looking for a module named "Intro to Pages" in the Enable Modules Menu.  Given that the directory for this exercise is `intro_to_pages`, I thought to update the exercise and module name to "Intro to Pages".  That also matches the other exercises.